controllers/activemqartemis_reconciler.go

```diff
@@ -1364,8 +1393,7 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) updateRequestedResource(customR
 	if updateError = resources.Update(client, requested); updateError == nil {
 		reconciler.log.V(1).Info("updated", "kind ", kind, "named ", requested.GetName())
 	} else {
-
-		reconciler.log.Error(updateError, "updated Failed", "kind ", kind, "named ", requested.GetName())
+		reconciler.log.V(1).Error(updateError, "updated Failed", "kind ", kind, "named ", requested.GetName())
```

Comment:
If my understanding is correct then we should use `V(0).Info` because `V(1).Error` is still an error and the users see it also if zap-log-level is ERROR
```suggestion
		reconciler.log.V(0).Info(updateError, "updated Failed", "kind ", kind, "named ", requested.GetName())
```